### DISCUSSION TITLE
Allow the usage of the osu key as mouse click in menus

### DISCUSSION THREAD

**Discussion by tbvns** [ppy/osu#30341] [Ideas] [answered]
It would be really cool for tablet user to be able to press [Osu key 1] to perform left click and [Osu key 2] to perform right click.
I'm talking about those two keys:
![image](https://github.com/user-attachments/assets/46af0689-c7b6-4ca9-b8bb-2de7068872d4)

**Comment by Tenexxt** [comment]
In osu you can turn off the mouse buttons mid gameplay, but this doesn't apply to anything beside it, so I don't see where this would be useful.
Also worth mentioning that since in a lot of menus you search automatically (aka you can type already and you don't to have to click on the search Pannell first) this would break because the game would have to either completely disable the selected buttons or make it search and click at the same time, but I can see this getting annoying since you could click on the things you don't want to.

I could see the ability to right click being useful, but from what I heard it's getting worked on so you can use tablet buttons to bind things like that in game and not use external drivers, so this also will get replaced in some time.

**Comment by peppy** [comment, marked as answer]
Been requested hundreds of times, we'll eventually make it work.

**Reply by tbvns** [reply]
Ok, ty
